src/Spotflow.InMemory.Azure.ServiceBus/Internals/MessagesStore.cs

```diff
@@ -244,11 +242,11 @@ private ServiceBusReceivedMessage FinishReceiveMessageUnsafe(EnqueuedServiceBusM
               contentType: message.ContentType,
               enqueuedTime: timeProvider.GetUtcNow(),
               properties: message.ApplicationProperties,
+              subject: message.Subject,
```

Comment:
Could you please also forward `ReplyTo` and `PartitionKey` properties?

I have prepared a test for this: https://github.com/spotflow-io/in-memory-azure-test-sdk/blob/4c14686ba14a55c30416890e208960ac3730546e/tests/Tests/ServiceBus/ServiceBusReceiverTests.cs#L270

In this test, currently the assertions for `Subject`, `ReplyTo` and `PartitionKey` are commented out and they should be uncommented as a part of your PR.

Reply:
Of course. I added `PartitionKey` and `ReplyTo` and push the changes.